src/message_buffer.cpp

```diff
@@ -86,6 +88,7 @@ MessageBuffer::operator=(MessageBuffer&& other)
   _buffer = std::move(other._buffer);
 }
 
+// clang-format off
```

Comment:
Disabling this will cause issues with future changes where folks use tabs instead of spaces, etc.  Might be good that we update formatting instead of disabling it.

Reply:
This is specifically so it doesn't mess with the swap_bytes which was laid out very nicely by Paul. Figured I'd fix it while I was here (since clang-format so rudely broke it when I added it). It is turned back on almost immediately.